src/components/screens/admin-reply-screen.tsx

```diff
@@ -72,8 +72,8 @@ export class AdminReplyScreen extends React.Component<Props, State> {
         loading: true
       });
 
-      const metaformsApi = Api.getMetaformsApi(this.props.adminToken);
-      const repliesApi = Api.getRepliesApi(this.props.adminToken);
+      const metaformsApi = Api.getMetaformsApi(this.props.signedToken);
+      const repliesApi = Api.getRepliesApi(this.props.signedToken);
```

Comment:
Create const of signedToken from props at the start of function as it is referenced multiple times